docs/tailwind-v4-migration.md

```
@@ -0,0 +1,713 @@
+| Navigation | Smooth scroll works | [ ] |
+| Animations | All Framer Motion animations work | [ ] |
+| Contact form | Form submits correctly | [ ] |
+| Responsive design | Layouts adapt correctly | [ ] |
+| Companies slider | Carousel works | [ ] |
+| Experience timeline | Renders correctly | [ ] |
+
+### Visual Checks
+
+| Element | Light Mode | Dark Mode |
+|---------|------------|-----------|
+| Header background | Semi-transparent white | Semi-transparent dark | [ ] |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)